### ISSUE TITLE
Dark Mode [feature request] 

### ISSUE THREAD

**Issue by Sportsgeist** [mbruel/ClementineRemote#7]
App works flawlessly for me. The only thing missing is Dark Mode. At night it's really bright, especially on a tablet. Really awesome would be two kinds. One for Oled Screens with true black backgrounds and one for LCDs with not quite true blacks.

**Comment by mbruel** [comment]
I'll think about it for the next release but I might need two versions for all the buttons...

**Comment by Sportsgeist** [comment]
That would be awesome!

And as far as I know, the rule with open source is: Never ask for a release date.
So, take as long as you need and thank you in advance.

Sportsgeist

**Comment by mbruel** [comment]
yep, I'll do when I've some time, hopefully by the end of the summer or the year ;)
out of curiosity, on which devise do you use it?
are you happy with it? any other suggestions?

**Comment by Sportsgeist** [comment]
It runs on a Samsung Tab S2 T719 running Lineage 17.1. Installed with Aurora (sorry, no Google my devices). And on my Samsung Note 4 running LineageOS 18.1
I've not yet tried it on my 1.Gen Moto G with half broken NandFlash running an ancient Paranoid Android, but still works as a RemoteControl and still has awesome battery life.

On the last point of suggestions. I'll sit down and write down some thoughts and ideas.
Do you want it as a new thread or rather via gmail? Might take me some days.

**Comment by mbruel** [comment]
> Installed with Aurora (sorry, no Google my devices).

I don't know Aurora but you could use FDroid too.


>  Do you want it as a new thread or rather via gmail? Might take me some days.

we can discuss here and then yeah later it would be best to create one request per feature.

About your usage of Clementine, do you use the version I've patched or the regular 1.3.1? Do you use the Files Tab on the remote?
